src/components/AboutUs/TheBottomLineSection.js

```diff
@@ -28,8 +28,10 @@ const useStyles = makeStyles((theme) => ({
         fontSize: '22px',
         lineHeight: '38px',
         fontWeight: 600,
+        paddingBottom: 50, 
         textDecoration: 'underline',
         textDecorationColor: '#FF7500',
+        textUnderlinePosition: 'under',
         [theme.breakpoints.down('sm')]: {
             margin: '0px 20px'
         }
```

Comment:
On the smallest mobile screen (320px) the cheeto is not on the same line as the title, change font size to 18px to fix the problem
<img width="203" alt="Screen Shot 2020-10-16 at 7 48 36 PM" src="https://user-images.githubusercontent.com/56731719/96322832-a4676180-0fe8-11eb-9852-9450be57de95.png">

Reply:
thanks for the catch! fixed